data-manager/jdbc/src/test/java/org/orbisgis/orbisdata/datamanager/jdbc/JdbcDataSourceTest.java

```diff
@@ -867,6 +867,159 @@ void testGetDataSet() {
         assertTrue(dataset instanceof ITable);
     }
 
+    /**
+     * Test the {@link JdbcDataSource#getTableNames(String)},
+     * {@link JdbcDataSource#getTableNames(String, IJdbcDataSource.TableType...)},
+     * {@link JdbcDataSource#getTableNames(String, String)},
+     * {@link JdbcDataSource#getTableNames(String, String, IJdbcDataSource.TableType...)},
+     * {@link JdbcDataSource#getTableNames(String, String, String)},
+     * {@link JdbcDataSource#getTableNames(String, String, String, IJdbcDataSource.TableType...)} methods.
+     */
+    @Test
+    void getTableNameTest() {
+        assertTrue(ds1.getTableNames("toto").isEmpty());
+        assertTrue(ds2.getTableNames("toto").isEmpty());
```

Comment:
Would be nice to add a test on a real postgresql database
For example in postgres you have the following types
Type of the table: BASE TABLE for a persistent base table (the normal table type), VIEW for a view, FOREIGN TABLE for a foreign table, or LOCAL TEMPORARY for a temporary table